config/initializers/clearance.rb

```diff
@@ -1,5 +1,5 @@
 Clearance.configure do |config|
-  config.allow_sign_up = ENV['DISABLE_SIGNUP'] ? false : true
```

Comment:
I have changed it to `(ENV['DISABLE_SIGNUP'].to_s == 'true') ? false : true`. I hope intended use is that `ENV['DISABLE_SIGNUP']` will set to `true` on staging instance.